python3.9/cookiecutter-aws-sam-step-functions-etl-python/{{cookiecutter.project_name}}/functions/etl_check/app.py

```diff
@@ -0,0 +1,69 @@
+import boto3
+import json
+from datetime import datetime
+
+session = boto3.session.Session()
+client = session.client("glue")
+
+class DateTimeEncoder(json.JSONEncoder):
+    def default(self, o):
+        if isinstance(o, datetime):
+            return o.isoformat()
+        return json.JSONEncoder.default(self, o)
+
+def lambda_handler(event, context):
+    """Sample pure Lambda function
```

Comment:
Should this comment be updated to explain what this method/lambda function is doing?